login-fire-form.html

```diff
@@ -263,6 +263,33 @@
       },
 
       /**
+       * Current value of the email field.
```

Comment:
The Polymer linter was complaining about the following attributes declarations because they were missing.